### PR TITLE
Issue 133 social network add read.md

### PR DESCRIPTION
**Pull request by davetownG**
Issue 133

Created README.md for Social Network Schema example. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: schema_design/SchemaExamples/SocialNetwork/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+- Sort Key (SK): Contains the ID of an entity in the Partition Key collection 
+    **or** 
+  a descriptor of the attributes ("count", "info") for the primary key of \<PK\>\<SK\>
+
+    - Examples:  
+
+      | PK | SK | Sample Attributes |
+      | ----------- | ----------- | ----------- |
+      | u#12345 | "count" | follower#, following#, post# |
```
[the remark this answers is not in the document]
Comment (davetownG) [reply schema_design/SchemaExamples/SocialNetwork/README.md]: Update completed

**Comment by davetownG** [comment]
Added Hyperlink to Social Media json file.

File: schema_design/SchemaExamples/SocialNetwork/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (davetownG) [reply schema_design/SchemaExamples/SocialNetwork/README.md]: Hi @tebanieo, do I need to do another pull request for this? Issues are fixed. Thanks

**Comment by davetownG** [comment]
Hi @tebanieo, do I need to do another pull request for this? Issues are fixed. Thanks

**Comment by tebanieo** [comment]
This PR completes the issue #133.

**Pull request merged**